model/src/test/java/org/cloudfoundry/identity/uaa/oauth/jwk/JsonWebKeyTest.java

```diff
@@ -0,0 +1,102 @@
+package org.cloudfoundry.identity.uaa.oauth.jwk;
```

Comment:
renamed this class in https://github.com/cloudfoundry/uaa/pull/2116
So depending on what PR comes in first we can change it